Civi/Api4/Generic/AutocompleteAction.php

```diff
@@ -470,8 +476,8 @@ private function inputMatchesDataType(): bool {
     if (!strlen($this->input) || \CRM_Utils_Rule::integer($this->input)) {
       return TRUE;
     }
-    $currentSearchField = $this->getField($this->getCurrentSearchField());
-    return $currentSearchField['data_type'] !== 'Integer';
+    $currentSearchField = $this->getField($this->getCurrentSearchField())['data_type'] ?? '';
```

Comment:
reason is for api_entity = EntitySet even if we get the searchField, like here `label`, we cannot fetch its metadata and this causes a test failure : 
```
Trying to access array offset on null

/home/homer/buildkit/build/build-0/web/core/Civi/Api4/Generic/AutocompleteAction.php:480
```